llm/dbrx/README.md

```diff
@@ -0,0 +1,177 @@
+I 03-27 21:08:53 optimizer.py:906] -----------------------------------------------------------------------------------------------------
+I 03-27 21:08:53 optimizer.py:906]  CLOUD   INSTANCE               vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE     COST ($)   CHOSEN
+I 03-27 21:08:53 optimizer.py:906] -----------------------------------------------------------------------------------------------------
+I 03-27 21:08:53 optimizer.py:906]  AWS     p4d.24xlarge[Spot]     96      1152      A100:8         us-east-2b      4.13          ✔
+I 03-27 21:08:53 optimizer.py:906]  GCP     a2-ultragpu-4g[Spot]   48      680       A100-80GB:4    us-east4-c      7.39
+I 03-27 21:08:53 optimizer.py:906]  GCP     a2-highgpu-8g[Spot]    96      680       A100:8         us-central1-a   11.75
+I 03-27 21:08:53 optimizer.py:906]  GCP     a2-ultragpu-8g[Spot]   96      1360      A100-80GB:8    us-east4-c      14.79
+I 03-27 21:08:53 optimizer.py:906]  GCP     a2-megagpu-16g[Spot]   96      1360      A100:16        us-central1-a   22.30
+I 03-27 21:08:53 optimizer.py:906] -----------------------------------------------------------------------------------------------------
+...
+```
+
+</details>
+
+> To run on K8s or use an on-demand instance, pass `--use-spot False` to the above command.
+
+Wait until the model is ready (this can take 10+ minutes), as indicated by these lines:
+```console
+...
+(task, pid=17433) Waiting for vllm api server to start...
+...
+(task, pid=17433) INFO:     Started server process [20621]
+(task, pid=17433) INFO:     Waiting for application startup.
+(task, pid=17433) INFO:     Application startup complete.
+(task, pid=17433) INFO:     Uvicorn running on http://0.0.0.0:8081 (Press CTRL+C to quit)
+...
+(task, pid=17433) Running on local URL:  http://127.0.0.1:8811
+(task, pid=17433) Running on public URL: https://xxxxxxxxxx.gradio.live
+...
+(task, pid=17433) INFO 03-28 04:32:50 metrics.py:218] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
+```
+:tada: **Congratulations!** :tada: You have now launched the DBRX Instruct LLM on your infra.
+
+You can play with the model via
+- Standard OpenAPI-compatible endpoints (e.g., `/v1/chat/completions`)
+- Gradio UI (automatically launched)
+
+To curl `/v1/chat/completions`:
+```console
+IP=$(sky status --ip dbrx)
+
+curl http://$IP:8081/v1/chat/completions \
+  -H "Content-Type: application/json" \
+  -d '{
+    "model": "databricks/dbrx-instruct",
+    "messages": [
+      {
+        "role": "system",
+        "content": "You are a helpful assistant."
+      },
+      {
+        "role": "user",
+        "content": "Who are you?"
+      }
+    ]
+  }'
+```
+
+To use the Gradio UI, open the URL shown in the logs:
+```console
+(task, pid=17433) Running on local URL:  http://127.0.0.1:8811
+(task, pid=17433) Running on public URL: https://xxxxxxxxxx.gradio.live
+```
```

Comment:
```suggestion
To use the Gradio UI, open the URL shown in the logs:
```console
(task, pid=17433) Running on public URL: https://xxxxxxxxxx.gradio.live
```
```
should we only include the gradio URL to reduce confusion